src/legacy/core_plugins/kibana/public/discover/np_ready/components/field_chooser/field_chooser.js

```diff
@@ -281,7 +280,7 @@ export function createFieldChooserDirective($location, config, $route) {
           });
         });
 
-        const fields = new FieldList(indexPattern, fieldSpecs);
+        const fields = indexPattern.fields;
```

Comment:
@kertal you see any reason why this wouldn't be a reasonable replacement?

Reply:
can't think of any 👍